### PR TITLE
Add persistence to /new

### PR DESCRIPTION
**Pull request by fatlotus**
N.B. This pull request is dependent on #57.

Largely, this just adds the .put() call and the key stuff I was talking about. I haven't done anything with the "place inquiry" form, but that can come next.

### CODE REVIEW

File: caravel/controllers/forms.py [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
-    price = DecimalField('Price', places=2)
-    description = TextAreaField('Description')
-    images = FieldList(FormField(ImageEntry), min_entries=5)
-    submit = SubmitField('Post')
\ No newline at end of file
+    title = StringField("Listing Title",
+                validators=[DataRequired()])
+    seller = StringField("Email", description="UChicago Email Required",
+                 validators=[Email()])
+    price = DecimalField("Price", places=2)
```
Comment (georgeteo) [inline caravel/controllers/forms.py]: Do we want to require price? Free should be 0.00 entered?

File: caravel/controllers/forms.py [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
-    title = StringField('Listing Title')
-    email = StringField('Email (UChicago email preferred)', validators=[Email()])
-    price = DecimalField('Price', places=2)
-    description = TextAreaField('Description')
-    images = FieldList(FormField(ImageEntry), min_entries=5)
-    submit = SubmitField('Post')
\ No newline at end of file
+    title = StringField("Listing Title",
+                validators=[DataRequired()])
+    seller = StringField("Email", description="UChicago Email Required",
```
Comment (georgeteo) [inline caravel/controllers/forms.py]: Not required?

File: caravel/controllers/forms.py (hunk 1 above)
Comment (fatlotus) [inline caravel/controllers/forms.py]: Sure! I've added validation to this field.

File: caravel/controllers/forms.py (hunk 2 above)
Comment (fatlotus) [inline caravel/controllers/forms.py]: I believe the `validate_seller` function takes care of this.

**Comment by fatlotus** [comment]
Bump! Merge conflicts fixed.

**Pull request merged**